BridgeCareApp/BridgeCare/Controllers/PerformanceController.cs

```diff
@@ -22,8 +22,9 @@ public PerformanceController(IPerformance performanceInterface, BridgeCareContex
         /// Get: api/performance
         /// </summary>
         [ModelValidation("Given simulation data is not valid")]
+        [Route("api/GetPerformance/{simulationId}")]
```

Comment:
snuck this change performance api into this branch, makes api easier to call

Reply:
ok